server/api.js

```diff
@@ -89,10 +101,192 @@ router.get("/userList", async (req, res) => {
 	res.status(200).json(fetchUserList);
 });
 
+const fetchAllData = async (startDate) => {
+	const channelList = await getChannelList();
+	const result = channelList.channels.map(async (channel) => {
+		const channelId = channel.id;
+		const data = await getChannelHistory(channel.id, startDate, " ");
+		const extractedData = data.messages.map((message) => {
+			const userId = message.user;
+			const type = message.type;
+			const date = message.ts;
+			const day = new Date(date * 1000).toLocaleDateString();
+			const threadDate = message.thread_ts ? message.thread_ts : false;
+			const reaction = message.reactions ? message.reactions : false;
+			const replyUsers = message.reply_users ? message.reply_users : false;
+			return {
+				channelId,
+				userId,
+				type,
+				date,
+				day,
+				threadDate,
+				reaction,
+				replyUsers,
+			};
+		});
+		return [...extractedData];
+	});
+	return Promise.all(result);
+};
+
+router.get("/dailyStatistic", async (req, res) => {
```

Comment:
I feel the dailyStatistic page storing data (I'm assuming it stores due to insertDataToTable) is a bit unexpected.